.github/workflows/changeset.yaml

```diff
@@ -31,22 +32,15 @@ jobs:
   notify-slack:
     runs-on: ubuntu-latest
     needs: version
+    if: needs.version.outputs.hasChangesets == 'false'
     env:
       PUBLISHED: ${{ needs.version.outputs.hasPublished }}
     steps:
-      - name: Publish successful
-        if: ${{ env.PUBLISHED == 'true' }}
+      - name: Send
+        if:
         uses: slackapi/slack-github-action@v1.24.0
         with:
           channel-id: "C02NUQ27G56"
-          slack-message: "Packages have been published"
-        env:
-          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
-      - name: Publish failed
-        if: ${{ env.PUBLISHED == 'false' }}
-        uses: slackapi/slack-github-action@v1.24.0
-        with:
-          channel-id: "C02NUQ27G56"
-          slack-message: "Publish failed"
+          slack-message: ${{ env.PUBLISHED == 'true' && 'Packages have been published' || 'Publishing failed' }}
```

Comment:
This is interesting... Is this the equivalent of a ternary? 😂

Reply:
Yes! Haha it's super weird